src/MSBuild.Bootstrap.Utils/MSBuild.Bootstrap.Utils.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build.Utilities.Core" VersionOverride="15.5.180" ExcludeAssets="runtime" PrivateAssets="all" />
```

Comment:
Just curious - why this specific version?

Reply:
the same version was taken as in https://github.com/dotnet/msbuild/blob/07be3a17b2459920aeca63592fa4b32aeab10629/src/Samples/PortableTask/PortableTask.csproj#L15

it's needed just for access to basic msbuild API, so we don't expect to have any changes here

Reply:
And can it possibly be a ProjectReference?